website/src/App.js

```diff
@@ -0,0 +1,47 @@
+import React, { Suspense } from 'react';
+import { useState } from 'react';
+import './App.css';
+const Cardui = React.lazy(() => import('./Cardui'));
+// import * as data from 'https://raw.githubusercontent.com/Open-Source-Community-VIT-AP/OpenSource101/master/.data/complete-data.json';
+
+function App() {
+  // console.log(data);
+
+    const [data, setdata] = useState([]);
+
+  const getapi = () =>{
+
+    fetch('https://raw.githubusercontent.com/Open-Source-Community-VIT-AP/OpenSource101/master/.data/complete-data.json')
+    .then((response) => response.json())
+    .then((json) => {
+      setdata(json);
+      
+    })
+  }
+  getapi();
+  return (
+    <div className="App">
+      <h1>Awesome Contribs</h1>
```

Comment:
Updated header to `Awesome Contributors`

Reply:
done